doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2291,26 +2240,19 @@ the ``Generalizable`` vernacular command to avoid unexpected
 generalizations when mistyping identifiers. There are several commands
 that specify which variables should be generalizable.
 
-.. cmd:: Generalizable All Variables
+.. cmd:: Generalizable {| All Variables | No Variables | {| Variable | Variables } {+ @ident } }
 
-   All variables are candidate for
+   All variables are candidates for
    generalization if they appear free in the context under a
    generalization delimiter. This may result in confusing errors in case
    of typos. In such cases, the context will probably contain some
    unexpected generalized variable.
 
-.. cmd:: Generalizable No Variables
-
-   Disable implicit generalization  entirely. This is the default behavior.
-
-.. cmd:: Generalizable {| Variable | Variables } {+ @ident }
+   The :n:`No Variables` option disables implicit generalization entirely.  This is the
+   default behavior.
 
-   Allow generalization of the given identifiers only. Calling this command multiple times
-   adds to the allowed identifiers.
-
-.. cmd:: Global Generalizable
```

Comment:
The new description should say that the `global` attribute is supported.

Reply:
Yeah, take a look at what I did.